### PR TITLE
Feature/team1 task2 change in tax calculation for businessmen working on the simplified taxation

### PR DESCRIPTION
**Pull request by altaras0v**

### CODE REVIEW

File: src/com/company/model/AccountIESimple.java [hunk 1]
```diff
@@ -5,6 +5,8 @@
 public class AccountIESimple extends Account {
     public AccountIESimple(Integer id, BigDecimal sumValue) {
         super(id, sumValue);
-        setTaxRate(5);
```
Comment (LStanislav) [inline src/com/company/model/AccountIESimple.java:8]: Are you shure&

File: src/com/company/Main.java [hunk 2]
```diff
@@ -8,14 +8,17 @@
...
+//
         DataProvider dataProvider = new DataProvider();
 
         Account painter = dataProvider.getPainter();
         AccountIE company = new AccountIE();
         Account ieSimple = dataProvider.getIeSimple();
+        ieSimple.setInPreferentialTaxZone(true);
         ieSimple.setPayments(dataProvider.generateIESimplePayments());
 
+
```
Comment (LStanislav) [inline src/com/company/Main.java:20]: why is it clear

File: src/com/company/model/AccountIESimple.java (hunk 1 above)
Comment (altaras0v) [reply src/com/company/model/AccountIESimple.java:8]: оф корсе

File: src/com/company/Main.java (hunk 2 above)
Comment (altaras0v) [reply src/com/company/Main.java:20]: ok,fix!

**Pull request merged**